src/kibana/plugins/vis_types/controls/line_interpolation_option.html

```diff
@@ -0,0 +1,7 @@
+<div>
+  <label>
+    Line Interpolation
```

Comment:
Can we `s/Interpolation/Smoothing` this? I don't think most people know what interpolation is. Also, if there's only on or off (linear or smooth) and we don't anticipate adding more options than that why not just make it a checkbox?